src/swapi_client/v2/models/core.py

```
@@ -0,0 +1,52 @@
+    /api/settings
+    /api/modules
+    /api/user/profile
+    itd.
+    """
+
+    client = None  # identycznie jak Commission.client
+
+    @classmethod
+    def objects(cls) -> CoreQuerySet:
+        if cls.client is None:
+            raise RuntimeError("Core.client must be assigned with SWAPIClient.")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)